### ISSUE TITLE
Device Endpoint Support

### ISSUE THREAD

**Issue by DanielDarrenJones** [printu/customerio#16]
Hi, do you have any plans to support the devices API endpoint?

**Comment by krzaczek** [comment]
@DanielDarrenJones sure it should be ease to implement. Should be ready till monday.

**Comment by krzaczek** [comment]
https://customer.io/docs/api/#apitrackcustomersdevices_update

**Comment by DanielDarrenJones** [comment]
@krzaczek Thanks for getting back so fast, would really appreciate that, the package looks great!

**Comment by krzaczek** [comment]
```php
$client->customers->devices->add([ // or update
        'id' => 12312,
        'device_id' => '2312321',
        'platform' => 'ios',
        'last_used' => 1514764800; //optional
]);

$client->customers->devices->delete([
        'id' => 12312,
        'device_id' => '2312321',
]);
```

**Comment by krzaczek** [comment]
implemented in 2.2.3
